pysat/tests/test_orbits.py

```diff
@@ -1,3 +1,5 @@
+"""Test the pysat routines for the orbits class."""
```

Comment:
Recommend adding the license info to the head of the file (check one of the recent tests I updated).